response_operations_ui/static/css/components/nav.css

```diff
@@ -63,30 +81,53 @@
     font-weight: 400;
 }
 
-.navigation-tabs__tab--active .tab-label{
+.nav-tabs__tab--active .tab-label{
     font-weight: 600;
     border: 1px solid #666;
     background: #fff;
     border-bottom: 1px solid #fff;
 }
 
-.navigation-tabs__tab .tab-label{
+.nav-tabs__tab .tab-label{
     text-decoration: none;
 }
 
-.navigation-tabs__tab .tab-label:hover,
-.navigation-tabs__tab .tab-label:focus{
+.nav-tabs__tab .tab-label:hover,
+.nav-tabs__tab .tab-label:focus{
     text-decoration: underline;
     border: 1px solid #666;
     background-color: #4263c2;
+    /*background-color: #f7f7f7;*/
 }
 
-.navigation-tabs__tab .tab-label:hover{
+.nav-tabs__tab .tab-label:hover{
   text-decoration: underline;
   background-color: #4263c2;
   color: #fff;
 }
 
-.navigation-tabs__tab.create{
+
+
+.nav-tabs__tab.create{
   float: right;
 }
+
+.tab-label.create-new-message{
```

Comment:
With these new ones, I don't think they're being used anywhere on the site at the moment.  Would it be better to add them down the line to keep these files as clean as possible?